### PR TITLE
Added Image Upload

### PR DESCRIPTION
**Pull request by prithweedas**
* Added Multer
* configured storage 
* modified Item Model
* modified POST:/ items route

### CODE REVIEW

**Comment by prithweedas** [comment]
Sorry it took some time to understand multer myself

File: src/routes/items.js [hunk 1]
```diff
@@ -32,28 +33,42 @@ router.get("/:id", async (req, res, next) => {
...
-  console.log(userId);
-  try {
-    const result = await item.save();
-    res.status(201).json({
-      ok: true,
-      item: result
+router.post(
+  "/",
+  upload.single("image"),
+  authenticate,
```
Comment (ritwickdey) [inline src/routes/items.js]: first `upload.single("image")` and then `authenticate` middle-ware?

Comment (prithweedas) [reply src/routes/items.js]: that was a mistake refactored a bit more check them

**Comment by ritwickdey** [comment]
👍 👍 👍

**Pull request merged**